extensions/amp-share-tracking/0.1/amp-share-tracking.js

```diff
@@ -163,4 +163,8 @@ export class AmpShareTracking extends AMP.BaseElement {
   }
 }
 
-AMP.registerElement('amp-share-tracking', AmpShareTracking);
+
+// Install the extension.
+AMP.extension('amp-share-tracking', AMP => {
```

Comment:
Yes. I realized that just deferring side effects is not enough. I also need to later install them to the *right* space :)